src/modules/chat_highlight_blacklist_keywords/index.js

```diff
@@ -14,10 +14,10 @@ const REPEATING_SPACE_REGEX = /\s\s+/g;
 
 const BLACKLIST_KEYWORD_PROMPT = `Type some blacklist keywords. Messages containing keywords will be filtered from your chat.
 
-Use spaces in the field to specify multiple keywords. Place {} around a set of words to form a phrase, <> inside the {} to use exact search, and () around a single word to specify a username. Wildcards (*) are supported.`;
+Use spaces in the field to specify multiple keywords. Place {} around a set of words to form a phrase, <> inside the {} to use exact search, and () around a single word to specify a username. '$@' for all mentions, '$#' for all links. Wildcards (*) are supported.`;
```

Comment:
is there a reason not to combine these with wildcards. could we instead do `https://*` for all links, and `@*` for all mentions?

Reply:
ok I realize now that `http*` works, however I know that `@*` does not work because this filtering is done after the text is parsed into the span elements (text, mention, link, emote)